### PR TITLE
Provide information on how to disable crontriggers on all st2s

### PR DESCRIPTION
**Pull request by puneetk**
from @codyarray

### CODE REVIEW

File: docs/source/reference/ha.rst [hunk 1]
```diff
@@ -348,6 +349,14 @@ above support the capbility of being turned on-off individually therefore each b
 
 9. Use Nginx config for the blueprint boxes from `here <https://gist.github.com/manasdk/fce14029900e533a385d#file-st2_Nginx-conf>`__. In this config Nginx will act as the SSL termination endpoint for all the REST endpoints exposed by ``st2api``, ``st2auth`` and ``mistral-api``.
 
-10. See :doc:`/reference/sensor_partitioning` to decide on how to partition sensors that suit your requirements.
+10. To use crontiggers with mistral please do the following to only enable them on one server by doing the following in `/etc/st2/st2.conf`
```
Comment (manasdk) [inline docs/source/reference/ha.rst]: s/crontiggers/Timer triggers.

no need to call out mistral specifically as this is applicable even without mistral in the picture.

Comment (puneetk) [inline docs/source/reference/ha.rst]: Fixing.

File: docs/source/reference/ha.rst [hunk 2]
```diff
@@ -348,6 +349,14 @@ above support the capbility of being turned on-off individually therefore each b
 
 9. Use Nginx config for the blueprint boxes from `here <https://gist.github.com/manasdk/fce14029900e533a385d#file-st2_Nginx-conf>`__. In this config Nginx will act as the SSL termination endpoint for all the REST endpoints exposed by ``st2api``, ``st2auth`` and ``mistral-api``.
 
-10. See :doc:`/reference/sensor_partitioning` to decide on how to partition sensors that suit your requirements.
+10. To use crontiggers with mistral please do the following to only enable them on one server by doing the following in `/etc/st2/st2.conf` 
+
+    .. code-block:: yaml
+        
+        [timer]
+        enable = {{ True if ansible_hostname == groups['stackstorm'][0] else False }}
```
Comment (manasdk) [inline docs/source/reference/ha.rst]: I guess we should probably only call it out as 

```
[timer]
enable = False
```

**Comment by manasdk** [comment]
LGTM. Will merge after update.

**Comment by manasdk** [comment]
@puneetk Thanks!

**Pull request merged**